### PR TITLE
feat: add routing to Login page

### PR DESCRIPTION
**Pull request by AdrianValdes**

### CODE REVIEW

File: src/webapp/src/Navbar.tsx [hunk 1]
```diff
@@ -20,12 +21,20 @@ const Header = styled.div`
...
-      <Button size='medium' onClick={() => {}}>
-        Login
-      </Button>
-    </Header>
-  </HeaderStyles>
-);
+export const Navbar = () => {
+  const history = useHistory();
+  return (
+    <HeaderStyles>
```
Comment (paschalidi) [inline src/webapp/src/Navbar.tsx:27]: Looks like there is a border missing on the bottom of the navbar. Mind taking a moment in this PR to add it?
image

File: src/webapp/src/Navbar.tsx [hunk 2]
```diff
@@ -20,12 +21,20 @@ const Header = styled.div`
...
+      <Header>
+        <Button
```
Comment (paschalidi) [inline src/webapp/src/Navbar.tsx:29]: Also we want this button to be disabled when the user is in the login page

File: src/webapp/src/auth/Login.tsx [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+export const Login = () => (
```
Comment (paschalidi) [inline src/webapp/src/auth/Login.tsx:3]: These are the authentication options page to me. Perhaps we can call it `AuthenticationOptions`?

File: src/webapp/src/App.tsx [hunk 4]
```diff
@@ -1,22 +1,31 @@
...
+      <Route exact path='/'>
+        <Auth />
+        <Person />
+        <Button onClick={() => {}} type='secondary'>
+          Invite Team
+        </Button>
+        <Button onClick={() => {}} type='primary'>
+          New Project
+        </Button>
+        <Characters />
```
Comment (paschalidi) [inline src/webapp/src/App.tsx:27]: Can we get rid of those in this PR please?

**Review by paschalidi — COMMENTED** [review]
Overall looks good, minor comments only.

**Comment by paschalidi** [comment]
Also lets put the Navbar in a new directory called `components`, what you think? In that file lets follow the same structure as the one in storybook. Aka `Navbar` folder which will contain styles tests and the component itself. What you think?
image

File: src/webapp/src/Navbar.tsx (hunk 1 above)
Comment (AdrianValdes) [reply src/webapp/src/Navbar.tsx:27]: The border appear and disappear in the home route depending on the position of the scroll bar. In the login routes is where is always there. I have to find a way of doing that nicely.

Comment (paschalidi) [reply src/webapp/src/Navbar.tsx:27]: We can start by always having it there and find a way to remove it only in certain cases

File: src/webapp/src/App.tsx (hunk 4 above)
Comment (AdrianValdes) [reply src/webapp/src/App.tsx:27]: I was hoping for you to say that! I hate there! :D

Comment (AdrianValdes) [reply src/webapp/src/App.tsx:27]: I should only remove them from the UI, right? 

Or do you want me to completely remove the components from the project as well?

**Comment by AdrianValdes** [comment]
> Also lets put the Navbar in a new directory called components, what you think? In that file lets follow the same structure as the one in storybook. Aka Navbar folder which will contain styles tests and the component itself. What you think?

I like a lot this structure, I think it is going to help us to be organized. Thanks!

File: src/webapp/src/auth/Login.tsx (hunk 3 above)
Comment (AdrianValdes) [reply src/webapp/src/auth/Login.tsx:3]: Done

File: src/webapp/src/App.tsx (hunk 4 above)
Comment (paschalidi) [reply src/webapp/src/App.tsx:27]: If you think we might need them again lets keep them. But I doubt it. However feel free to remove them in another PR.

Comment (AdrianValdes) [reply src/webapp/src/App.tsx:27]: Yes, I will do it in another PR better.

**Pull request closed without merging**